corehq/apps/reports/standard/forms/reports.py

```diff
@@ -56,13 +56,24 @@ def submitfilter(self):
             self._submitfilter = SubmissionTypeFilter.get_filter_toggle(self.request)
         return self._submitfilter
 
+    @property
+    def sort_params(self):
+        sort_col_idx = int(self.request.GET.get('iSortCol_0', 2))
```

Comment:
This kind of index mapping feels a little brittle.  `sort_col` already defaults to `'received_on'` above, could `sort_prop` just fall back to `None` if unspecified or not found?

Reply:
Seems reasonable, since I added that `'received_on'` default to the esaccessor as well

1405e58